test/e2e/framework/providers/vsphere/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
this should be the same as https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/vsphere_volume/OWNERS
any reason they are different?

Reply:
Nice point. And is it nice to add @sandeeppsunny also into the file for maintaining it in the future?

Reply:
My bad. I have added all the names from the vsphere_volume OWNERS file and added my name as well to the list since I am maintaining the test code currently.

Reply:
Please add the content of https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/vsphere/OWNERS
to this file.

Reply:
Done